docs/platforms/javascript/common/sourcemaps/troubleshooting_js/debug-ids/index.mdx

```diff
@@ -0,0 +1,74 @@
+
+As a result, the JavaScript code running in production is typically not the same as the code written by the developer in the source editor. Instead, it has undergone several transformations, such as [minification](https://developer.mozilla.org/en-US/docs/Glossary/Minification), [transpilation](https://www.typescriptlang.org/), [downcompilation](https://babeljs.io/), [transformation](https://rollupjs.org/), [polyfilling](https://developer.mozilla.org/en-US/docs/Glossary/Polyfill), and others, aimed at improving performance and ensuring cross-browser compatibility.
+
+Stack traces are essential to debug errors, but stack traces that come from generated JavaScript are often unreadable or unusable! They look nothing like the code you wrote, you can’t connect them to your source code repository, and minified variables or function names make you lose context about what is happening.
+
+To ensure Sentry (and other tools) can provide stack traces that are both readable and useful, you’ll want to generate and use [source maps](https://firefox-source-docs.mozilla.org/devtools-user/debugger/how_to/use_a_source_map/index.html). Source maps are files that map the transformed JavaScript back to the original code. This allows Sentry to trace the error to your original source, which means you’ll get stack traces that look like the code you wrote, not the code you generated.
+
+To connect source maps to a JavaScript file, we relied on filenames, but these often were unreliable. For example, a new deploy of a source map with the same filename would mean the source mapping gets incorrectly applied.
+
+`//# source mappingURL=http://example.com/path/to/your/source map.map`
+
+Given the url-based filename approach was unreliable, we came up with [Debug IDs](/platforms/javascript/sourcemaps/debug-ids#what-is-a-debug-id).
+
+## What are Debug IDs
+
+Inspired by approaches in native language ecosystems, Debug IDs are globally unique ids that identify a transformed JavaScript file and its associated source map.
+
+![Debug IDs example](./img/debug-ids.png)
+
+When using sentry-cli or bundler plugins for Webpack, Vite, or Rollup, Debug IDs are deterministically generated based on the source file contents. Note that deterministic Debug IDs for esbuild are not currently supported.
+
+This approach is Sentry's recommended and most reliable method for associating minified JavaScript files with their original source code, making it easier to trace errors back to their source. As a result, **creating a release is no longer required**.
+
+<Alert>
+
+If you want to learn more about the rationale behind Debug IDs, we suggest taking a look at our [engineering blog](https://sentry.engineering/blog/the-case-for-debug-ids).
+
+</Alert>
+
+## How Sentry uses Debug IDs
+
+To use Debug IDs, you need **version 7.47** or higher of one of our [JavaScript SDKs](https://github.com/getsentry/sentry-javascript). Once you're using a supported SDK, the process works as follows:
```

Comment:
```suggestion
To use Debug IDs, you need **version 7.47.0** or higher of one of our [JavaScript SDKs](https://github.com/getsentry/sentry-javascript). Once you're using a supported SDK, the process works as follows:
```

Reply:
technically for nextjs at least i think v8 is required

Reply:
should we hide this info then? I got it from somewhere

Reply:
I wouldn't hide it, but maybe show it differently if the selected platform is Next.js. The nextjs sdk indeed supports sourcemaps from 8.0.0 onwards.

Reply:
yes but it can be that JavaScript is selected and it provides an overall info so I have to say about Next JS, right?

Reply:
nevermind